src/_pytest/python.py

```diff
@@ -1291,6 +1289,18 @@ def __init__(
         if keywords:
             self.keywords.update(keywords)
 
+        # todo: this is a hell of a hack
```

Comment:
Please add more information here, or create an issue and add a reference here. 👍